test/s3select_test.cpp

```diff
@@ -238,3 +238,33 @@ TEST(TestS3SElect, utcnow_function)
     ASSERT_EQ(s3select_res, boost::posix_time::to_simple_string(now));
 }
 
+void generate_csv(std::string& out, size_t size) {
+  // schema is: int, float, string, string
+  std::stringstream ss;
```

Comment:
I know this is boilerplate from other s3select functions--but it's actually pretty slow relative to (nowadays) std::fmt and a buffer of some kind ;)